packages/client/utils/LogRocketManager.ts

```diff
@@ -0,0 +1,70 @@
+import LogRocket from 'logrocket'
+
+interface User {
+  id: string
+  email: string
+}
+
+class LogRocketManager {
```

Comment:
As I mentioned above, this class allows deferring initialization to reduce the number of sessions (and, thus, the cost of LogRocket).

Now, in `AnalyticsPage`, we can call `setUser/unsetUser` whenever the user changes. Then, we can identify the proper user anytime the SDK is initialized.

Reply:
Sounds good - should I remove logrocket as part of this PR? Or should it be a follow-up?